README.md

```diff
@@ -1,46 +1,126 @@
+      - name: Local runner run
+        id: aws_start
+        uses: omsf-eco-infra/gha-runner@latest # This is pending publishing of this action
+        with:
+          repo: "<the repo name in terms of org/name, if not provided the current repo will be used>"
+          provider: "aws"
+          action: "start"
+          instance_count: <the number of instances, if not provided, defaults to 1>
+          aws_image_id: <the AWS image AMI to use for your runner>
+          aws_instance_type: <the AWS instance type>
+          aws_subnet_id: <the AWS subnet ID, will use default by default, optional>
+          aws_security_group_id: <the AWS security group ID, will use default by default, optional>
+          aws_iam_role: <the AWS IAM role to use, optional>
+          aws_region_name: <AWS region name, required>
+          aws_home_dir: <the ec2 instance home directory, for Amazon Linux 2023/Amazon Linux 2 this is /home/ec2-user>
+          aws_labels: <labels provided to AWS to be added to the GitHub runner>
+          # NOTE: aws_tags are provided as a list of key-values as seen below
+          aws_tags: >
+            [
+              {"Key": "Name", "Value": "test"}
+              {"Key": "Other", "Value": "item"}
+            ]
+        env:
+          GH_PAT: ${{ secrets.GH_PAT }} # Your GH PAT access token with enough scope to create runners
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }} # Your AWS access key id
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }} # You AWS secret access key
+      - name: Your test here
+        id: test
+        runs-on: self-hosted
+      - name: Stop instances
+        needs:
+          - aws_start
+          - test
+        if: ${{ always() }}
+        uses: omsf-eco-infra/gha-runner@latest # This is pending publishing of this action
+        with:
+          repo: "<the repo name in terms of org/name, if not provided the current repo will be used>"
+          provider: "aws"
+          action: "stop"
+          # NOTE: The output below is a JSON formatted mapping of AWS instance IDs to runner names. Below is the recommended usage.
+          instance_mapping: ${{ steps.aws_start.outputs.mapping }}
+          aws_region_name: <AWS region name, required>
+        env:
+          GH_PAT: ${{ secrets.GH_PAT }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
 
-Create a `.env` in the root of the project with the following environment variables:
-```sh
-AWS_ACCESS_KEY_ID="<Your AWS Access Key ID>"
-AWS_SECRET_ACCESS_KEY="<YOU AWS Secret Access Key>"
-GH_PAT="<A GitHub Personal Access Token>"
 ```
 
-For a breakdown on how to get these items please use the following [https://github.com/machulav/ec2-github-runner?tab=readme-ov-file#how-to-start](https://github.com/machulav/ec2-github-runner?tab=readme-ov-file#how-to-start).
+## Testing the action in development
```

Comment:
Somewhere in the README, you should include a list of the inputs from the actions.yml. As I recall, there are not automatically generated (which is a shame).